tests/src/test/java/alluxio/client/fuse/JNIFuseIntegrationTest.java

```diff
@@ -41,6 +41,6 @@ public void mountFuse(FileSystem fileSystem, String mountPoint, String alluxioRo
 
   @Override
   public void umountFuse(String mountPath) throws Exception {
-    mFuseFileSystem.umount();
+    mFuseFileSystem.umount(false);
```

Comment:
why false here? will it cause the test to leave the fuse process running while closing?

Reply:
done.